CHANGELOG.md

```diff
@@ -6,4 +6,6 @@ Initial Release
 - Added CHANGELOG.md
 - Added README.md
 - Added LICENSE
-- Added _sys_req_ewon_fw.mdx, _sys_req_java.mdx, _sys_req_maven.mdx
+- Added _sys_req_ewon_fw.mdx, _sys_req_java.mdx, _sys_req_maven.mdx, Configuration.mdx, _installation_or_upgrade.mdx
+- Added popover.js

```

Comment:
`highlight.js` is also available in the starter project.

Not sure what your thoughts are, but these common components should probably live in this repository as well. It allows for importing/selecting only the files necessary, and they are automatically kept up to date.

Reply:
Yeah, this is just a start. Anything that is common/generic and can or should be updated automatically can get added here.

Reply:
Agreed. It should be fairly simple to update the starter project to use this repository in the near future, and hopefully, that will be distributed to the connectors.